tools/run-tests.py

```diff
@@ -282,6 +284,11 @@ def iterate_test_runner_jobs(jobs, options):
 
         yield job, ret_build, test_cmd
 
+def generate_tests(options):
+    for filename in os.listdir("./tests/jerry/generators/"):
+        if filename.endswith(".py"):
+            execfile("./tests/jerry/generators/" + str(filename))
```

Comment:
As I mention, much better approach to use a python module. So, there will be no need to execute a script.